Dockerfile

```diff
@@ -20,23 +21,28 @@ RUN apk --no-cache add \
       linux-headers \
       make \
       openssl \
-    && cd /root \
+    && mkdir /zwave \
```

Comment:
Is this directory change really necessary? It is a breaking change from the examples so those would have to be updated as well.